cat.html

```diff
@@ -1,4 +1,9 @@
-<html>
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+<meta charset = "utf-8">
```

Comment:
This does not seem to have the correct indentation.
it should be something like:
```html
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset = "utf-8">
    ...
  </head>
```